code/_globalvars/misc.dm

```diff
@@ -5,20 +5,8 @@ GLOBAL_VAR_INIT(CELLRATE, 0.002)  // conversion ratio between a watt-tick and ki
 GLOBAL_VAR_INIT(CHARGELEVEL, 0.001) // Cap for how fast cells charge, as a percentage-per-tick (.001 means cellcharge is capped to 1% per second)
 
 // Announcer intercom, because too much stuff creates an intercom for one message then qdel()s it.
-GLOBAL_DATUM_INIT(global_announcer, /obj/item/radio/intercom, create_global_announcer())
-GLOBAL_DATUM_INIT(command_announcer, /obj/item/radio/intercom/command, create_command_announcer())
-
-// Load order issues means this can't be new'd until other code runs
-// This is probably not the way I should be doing this, but I don't know how to do it right!
-/proc/create_global_announcer()
-  spawn(0)
-    GLOB.global_announcer = new(null)
-  return
-
-/proc/create_command_announcer()
-  spawn(0)
-    GLOB.command_announcer = new(null)
-  return
+GLOBAL_DATUM_INIT(global_announcer, /obj/item/radio/intercom, new(null))
+GLOBAL_DATUM_INIT(command_announcer, /obj/item/radio/intercom/command, new(null))
```

Comment:
I tried to convert these, got runtimes on startup. I wish you the best of luck.

Reply:
Hm, I double checked again just now, and I didn't get any runtimes, but I'm on local so yknow.

Reply:
Ok so this actually runtimes in the DD window, not in the runtime viewer or logs. Yet AI arrival announcements worked fine when I tested them in game. But I'm probably going to revert this change, don't want to deal with this mess right now.